gateleen-core/src/main/java/org/swisspush/gateleen/core/json/transform/JoltSpecBuilder.java

```diff
@@ -13,20 +13,28 @@
  */
 public class JoltSpecBuilder {
 
+    /**
+     * See {@link JoltSpecBuilder#buildSpecWithMetadata(String, boolean)} with {@code false} for param withMetadata
+     */
+    public static JoltSpec buildSpec(String jsonSpec) throws JoltSpecException {
+        return buildSpecWithMetadata(jsonSpec, false);
+    }
+
     /**
      * Builds a {@link JoltSpec} object based on the provided string representation of the json spec.
      *
      * If an error occurs during the creation of the spec, a {@link JoltSpecException} is thrown.
      *
      * @param jsonSpec the string representation of the spec
+     * @param withMetadata does the json input to use the spec against has meta data
      * @return returns a {@link JoltSpec}
-     * @throws JoltSpecException when an error occured during spec parsing
+     * @throws JoltSpecException when an error occurred during spec parsing
      */
-    public static JoltSpec buildSpec(String jsonSpec) throws JoltSpecException {
+    public static JoltSpec buildSpecWithMetadata(String jsonSpec, boolean withMetadata) throws JoltSpecException {
```

Comment:
Suggest to drop `WithMetadata` from method name.

> // Example call
> buildSpecWithMetadata( foo , false );

IMHO that call sounds somewhat counter-intuitive.